test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -8,37 +8,32 @@ import { Application } from '../../../../../automation';
 export function setup() {
 	describe('Notebook', () => {
 
-		it('can open new notebook, configure Python, and execute one cell', async function () {
+		it('can perform basic text cell functionality', async function () {
 			const app = this.app as Application;
 			await app.workbench.sqlNotebook.newUntitledNotebook();
-			await app.workbench.sqlNotebook.addCell('code');
-			await app.workbench.sqlNotebook.waitForTypeInEditor('print("Hello world!")');
-			await app.workbench.sqlNotebook.waitForKernel('SQL');
-
-			await app.workbench.sqlNotebook.changeKernel('Python 3');
-			await app.workbench.configurePythonDialog.waitForConfigurePythonDialog();
-			await app.workbench.configurePythonDialog.installPython();
-			await app.workbench.sqlNotebook.waitForKernel('Python 3');
-
-			await app.workbench.sqlNotebook.runActiveCell();
-			await app.workbench.sqlNotebook.waitForActiveCellResults();
+			await app.workbench.sqlNotebook.addCellFromPlaceholder('Markdown');
+			await app.workbench.sqlNotebook.waitForPlaceholderGone();
 
-			await app.workbench.quickaccess.runCommand('workbench.action.revertAndCloseActiveEditor');
-		});
+			await app.code.dispatchKeybinding('escape');
+			await app.workbench.sqlNotebook.waitForDoubleClickToEdit();
+			await app.workbench.sqlNotebook.doubleClickTextCell();
+			await app.workbench.sqlNotebook.waitForDoubleClickToEditGone();
 
-		it('can open ipynb file, run all, and save notebook with outputs', async function () {
-			const app = this.app as Application;
-			await openAndRunNotebook(app, 'hello.ipynb');
+			await app.workbench.sqlNotebook.changeTextCellView('Split View');
+			const sampleText: string = 'Test text cells';
+			await app.workbench.sqlNotebook.waitForTypeInEditor(sampleText);
+			await app.code.dispatchKeybinding('escape');
+			await app.workbench.sqlNotebook.waitForTextCellPreviewContent(sampleText, 'p');
 		});
 
-		it('can open ipynb file from path with spaces, run all, and save notebook with outputs', async function () {
+		it('can perform basic code cell functionality', async function () {
 			const app = this.app as Application;
-			await openAndRunNotebook(app, 'helloWithSpaces.ipynb');
-		});
+			await app.workbench.sqlNotebook.newUntitledNotebook();
+			await app.workbench.sqlNotebook.addCellFromPlaceholder('Code');
+			await app.workbench.sqlNotebook.waitForPlaceholderGone();
 
-		it('can open ipynb file from path with escaped spaces, run all, and save notebook with outputs', async function () {
-			const app = this.app as Application;
-			await openAndRunNotebook(app, 'helloWithEscapedSpaces.ipynb');
+			const sampleText: string = 'SELECT * FROM sys.tables';
+			await app.workbench.sqlNotebook.waitForTypeInEditor(sampleText);
```

Comment:
Did you look into actually running the cell as well? Just typing is a good start but we really should be actually verifying that it does run and show stuff like the results and whatnot.

Reply:
Yep, originally I thought we had tests for running code cells but looking back now they are all running Python code cells so makes sense to test SQL code cells as well. I must've gotten mixed up with the query editor tests. I'll add that next week.